console-src/src/types/oAuthClient.tsx

```diff
@@ -1,16 +1,16 @@
 enum Scope {
-  OFFLINE = "offline",
-  ACCOUNTS = "accounts",
-  USERS = "users",
-  TRANSACTIONS = "transactions",
-  TRANSFERS = "transfers",
-  SUBSCRIPTIONS = "subscriptions",
-  STATEMENTS = "statements"
+  OFFLINE = "OFFLINE",
```

Comment:
we expose these as uppercase in our graph so lowercase was actually incorrect here

Reply:
You are right, `type-graphql` exposes the enum key - not its internal value.